src/serac/infrastructure/profiling.cpp

```diff
@@ -20,12 +20,6 @@ std::optional<cali::ConfigManager> mgr;
 }  // namespace
 #endif
 
-/**
- * @brief initialization for the profiling libraries (currently adiak and caliper, when enabled)
- *
- * @param comm the mpi communicator
- * @param options string of options passed to caliper, when enabled
- */
 void initialize([[maybe_unused]] MPI_Comm comm, [[maybe_unused]] std::string options)
```

Comment:
I'm not sure what's going on here. My doxygen_docs target tells me that `comm` is documented twice (once in the header, once in the .cpp file), so I deleted this redundant comment block. Now, my local doxygen build says this is correct, but the CI job complains that this function is not documented 😩

Reply:
Looks like this was caught by @adayton1 8 months ago, is this not working?

https://github.com/LLNL/serac/blob/a300c9c98c631d5503428aff9f67dbd2cdc0df90/scripts/azure-pipelines/docs_ignore_regexs.txt#L5-L6

Reply:
Confusingly, I didn't even change this source file but now the CI job passed. It's worrying that our CI jobs don't seem to have deterministic output.